disttask/loaddata/scheduler.go

```diff
@@ -101,9 +106,26 @@ func (s *ImportScheduler) SplitSubtask(ctx context.Context, bs []byte) ([]proto.
 }
 
 // OnSubtaskFinished implements the Scheduler.OnSubtaskFinished interface.
-func (s *ImportScheduler) OnSubtaskFinished(context.Context, []byte) error {
+func (s *ImportScheduler) OnSubtaskFinished(ctx context.Context, bs []byte) error {
 	logutil.BgLogger().Info("OnSubtaskFinished", zap.Any("taskMeta", s.taskMeta))
-	return nil
+	var subtaskMeta SubtaskMeta
+	if err := json.Unmarshal(bs, &subtaskMeta); err != nil {
+		return err
+	}
+
+	dataEngine, ok := s.dataEngines.Load(subtaskMeta.ID)
+	if !ok {
+		return errors.Errorf("data engine %d not found", subtaskMeta.ID)
+	}
+	engine, ok := dataEngine.(*backend.OpenedEngine)
+	if !ok {
+		return errors.Errorf("data engine %d not found", subtaskMeta.ID)
+	}
+	closedEngine, err := engine.Close(ctx)
+	if err != nil {
+		return err
+	}
+	return s.tableImporter.ImportAndCleanup(ctx, closedEngine)
```

Comment:
why the real work is put in `OnSubtaskFinished`? what will the subtask do in executing subtask?

Reply:
subtask executor read data and write to local disk, scheduler import the sst files